packages/cozy-flags/src/flag.js

```diff
@@ -74,7 +74,7 @@ const flag = function() {
 }
 
 export const listFlags = () => {
-  return store.keys()
+  return store.keys().sort()
```

Comment:
Should we take care of possible Uppercase flag?

Reply:
We do not have the case, I don't think it is a problem :)